nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+
+from tqdm.auto import tqdm
+
+from nemo.collections.asr.models.ctc_bpe_models import EncDecCTCModelBPE
+from nemo.collections.asr.models.ctc_models import EncDecCTCModel
+from nemo.utils import logging
+
+
+def transcribe_partial_audio(
+    asr_model,
+    path2manifest: str,
+    batch_size: int = 4,
+    logprobs: bool = False,
+    return_hypotheses: bool = False,
+    num_workers: int = 0,
+) -> List[str]:
+
+    assert (
+        type(asr_model) == EncDecCTCModelBPE or type(asr_model) == EncDecCTCModel
```

Comment:
You can do is instance(asr_model, EncDecCTCModel) - subclassing should work out of the box.